console: document useSecureCookies (#1673)
redpanda-data/documentation: docs/reference/console/config.mdx

  # already logged into Redpanda Console will be logged out and will have 
  # to log in again.
  jwtSecret: ""
  # The "useSecureCookies" option controls the storage of session cookies in 
  # Console. Enabling this option ensures that session cookies are stored 
  # with the secure attribute, allowing them to be sent only over HTTPS.

>   # If you are not using SSL in your environment, it is necessary to 

Reviewer: ```suggestion
  # If you are not using SSL in your environment, you must 
```
